tests/Application/config/routes/sylius_paypal.yaml

```diff
@@ -0,0 +1,12 @@
+sylius_paypal_admin:
```

Comment:
Such common routes file should be at `config/routes.yaml` to allow end-users to import all routes at once.